### ISSUE TITLE
Discord doesn’t support full markdown so blog posts will look wrong

### ISSUE THREAD

**Issue by SpyHoodle** [codedtogether/web#40]
**About the issue**
Discord announcements are done in markdown and currently the blogposts are in HTML. We’d have to have a markdown to HTML converter or just switch to markdown. Discord doesn’t support full markdown either. So you can’t have headings with `#` and stuff. The blog posts probably wouldn’t look right - so we’d have to find a fix. Also there should be a channel just for blog posts that takes a new post from the site and places it on here, ignoring markdown that isn’t supported like `#` or ignoring the HTML and just taking the plain text. Also we wouldn’t have headers then, so a `<h1>` or a `#` should convert the text into bold and underlined or something to make it stand out.

**Additional Context**
@taneltm has said that we write in markdown, but convert to HTML on the client side. This would have less chance for XSS to happen but we’d still have to be careful of headers with `#` and stuff, since discord does not support it.

**What this issue does**
Either one side would look wrong. Like headers would be just text, and not be any bigger than a paragraph.

**Comment by taneltm** [comment]
That's a really good point. There are a lot of markdown options that one would want to use in a blog post, like tables, links with names, image tags.. things that can't be used on Discord. So it would make sense to do "Discord to Blog" instead of "Blog to Discord".
There's also the thing that blog posts themselves have a separate title, category and tags.. metadata which we can't add in Discord - at least not so that it looks pretty... So "Discord to Blog" isn't that great either.

But what we could do is have a bot share an excerpt and a link to the blog post.
The excerpt should just be a block of text without fancy markdown.

**Comment by SpyHoodle** [comment]
Yeah, we’d need to just take only plain text using some sort of bot.

**Comment by abcdan** [comment]
Question, why would
>Discord to blog
make sense?

**Comment by SpyHoodle** [comment]
It was suggested by @anishanne, that new announcements get automatically posted to blog if a moderator reacts with a certain emoji. I guess it could make making blog posts easier if it was an announcement. 
Not my idea though.

**Comment by abcdan** [comment]
I don't think that would be the best, also we should rename the `Blog` to `Posts` as the plan was to also write articles for the website with the community. So only having it as a `Blog` could be a bad idea. 

I'm not sure what you guys think tho

**Comment by SpyHoodle** [comment]
Try not to bring up things that aren't related to this post, create a new issue and we can discuss opinions.

**Comment by abcdan** [comment]
It's related in a way, as it's also about the posts/blog system

**Comment by SpyHoodle** [comment]
Not about *this* specific issue. It's related in a way but going off-topic.
Create a *new* issue about it - its not the same at all and can fill up this issue with different stuff that makes the original issue neglected.
